src/coreclr/nativeaot/System.Private.CoreLib/src/System.Private.CoreLib.csproj

```diff
@@ -35,6 +35,11 @@
   <PropertyGroup Condition="'$(TargetsSingleThreadedWasm)' != 'true'">
     <FeaturePortableThreadPool>true</FeaturePortableThreadPool>
     <FeaturePortableTimer>true</FeaturePortableTimer>
+    <!-- LLVM-TODO: Remove FeatureWasmManagedThreads in favor of WasmEnableThreads. -->
+    <FeatureWasmManagedThreads Condition="'$(WasmEnableThreads)' == 'true'">true</FeatureWasmManagedThreads>
+    <FeatureWasmPerfTracing Condition="('$(TargetsBrowser)' == 'true' or '$(TargetsWasi)' == 'true') and ('$(WasmEnableThreads)' == 'true')">true</FeatureWasmPerfTracing>
```

Comment:
Is `FeatureWasmPerfTracing` actually used for anything? We don't really have any tracing support...

Reply:
Removed.